src/doc/gui_manual/Operators/OperatorTypes/PICS/Poincare_operator.rst

```diff
@@ -0,0 +1,318 @@
+   limits must be utilized.
+
+Puncture Plane
+    The typical puncture plane is the poloidal plane but in some cases seeing
+    the punctures in the toroidal plane may be useful.
+
+Note: when selecting “Toroidal” the “Analysis” must also be set to “Punctures
+only” as there is currently no analysis in the toroidal plane. See below.
+
+
+Analysis
+^^^^^^^^
+
+Analysis Type
+"""""""""""""
+
+The user may turn the analysis on or off. Options are:
+
+None - Puncture only
+    This will result in constructing a traditional Poincaré plot using only
+    points. 
+
+Full
+    This will analyze each fieldline’s geometry and attempt to reconstruct the
+    cross sectional profile of the surface which the fieldline lies on.
+    Further, the analysis attempt to identify the topology of the surface. 
+
+Maximum toroidal winding
+    Limit the search of the toroidal winding to lower order values. Zero
+    indicates no limit.
+
+Override toroidal winding
+    In some cases, such as debugging, it may be informative to force the
+    toroidal winding to have a set value. Zero indicates no override. 
+
+Override poloidal winding
+    In some cases such as debugging, it may be informative to force the
+    poloidal winding to have set value. Zero indicates no override.
+
+Winding pair confidence (Range 0-1, Default 0.9)
+    Sets the limit for the number of mismatches in the consistency in the
+    winding pairs.
+
+Detect Rational Surface
+    Allows for the construction of rational surfaces via an iterative process.
+
+    * Maximums iterations – sets the number of iterations for constructing the
+    rational surface. Typically, it can be found with 5-10 iterations. 
+
+Note: The rational surface construction is experimental code and does not
+always work!!!! 
+
+Detect O Points
+    Allows for the detection of O points in “island chains” via an iterative
+    process.
+    
+    * Maximum iterations - sets the number of iterations for locating the O
+      point. Typically, it can be found with 5 iterations. 
+
+Note: The critical point detection is experimental code and does not always
+work!!!! 
+
+
+Perform O-Line Analysis
+    Calculate the poloidal winding relative the O-Line (central axis) which
+    provides a more accurate winding value.
+
+    * O-Line toroidal windings (Default 1) – sets the toroidal winding value,
+      i.e. the period (for the central axis the period would be 1). 
+    * O-Line Axis Point File - allows the user to select a text file containing
+      the points along the axis from 0 to 360 degrees (note there is no overlap
+      P(0) != P(n)).
+
+Show chaotic fieldlines as points
+    Because chaotic fieldlines can not be classified, they are not displayed
+    unless this is checked. 
+
+Show islands only
+    Culls the results so that only island chains are displayed. 
+
+Show ridgelines
+    Displays the 1D plots of the distance and ridgeline samples.
+
+Verbose
```

Comment:
There is a lot of verbiage in these interface labels that I think could potentially require further explanation or, better yet, an on-line ref that describes them that we simply link to here. 

In other words, this integral curve/LCS application domain probably has a lot of lingo that those working in this discipline routinely use and instead of repeating their definitions in our documentation, we just need a place to go where their lingo is described.

Reply:
This verbiage is not defining the terms themselves but rather describing how the setting changes what will be visualized.